python/audit/tests/test_config_log.py

```diff
@@ -32,47 +33,42 @@ def setUp(self) -> None:
         super().setUp()
 
         self.config = ObjectConfig.objects.create(current=1, previous=1)
-        bundle = Bundle.objects.create()
-        prototype = Prototype.objects.create(bundle=bundle)
-        cluster = Cluster.objects.create(prototype=prototype, config=self.config)
+        self.bundle = Bundle.objects.create()
+        prototype = Prototype.objects.create(bundle=self.bundle)
+        self.cluster = Cluster.objects.create(prototype=prototype, config=self.config)
         ConfigLog.objects.create(obj_ref=self.config, config="{}")
         self.group_config = GroupConfig.objects.create(
             name="test_group_config",
-            object_id=cluster.pk,
+            object_id=self.cluster.pk,
             object_type=ContentType.objects.get(app_label="cm", model="cluster"),
             config_id=self.config.pk,
         )
 
-    def check_log(self, res: Response, log: AuditLog) -> None:
-        assert log.audit_object.object_id == res.data["id"]
-        assert log.audit_object.object_name == str(ConfigLog.objects.get(pk=res.data["id"]))
-        assert log.audit_object.object_type == AuditObjectType.Cluster
-        assert not log.audit_object.is_deleted
-        assert log.operation_name == "Cluster config log updated"
-        assert log.operation_type == AuditLogOperationType.Update
-        assert log.operation_result == AuditLogOperationResult.Success
-        assert isinstance(log.operation_time, datetime)
-        assert log.user.pk == self.test_user.pk
-        assert isinstance(log.object_changes, dict)
-
-    def check_log_denied(self, log: AuditLog) -> None:
-        assert not log.audit_object
-        assert log.operation_name == "config log updated"
-        assert log.operation_type == AuditLogOperationType.Update
-        assert log.operation_result == AuditLogOperationResult.Denied
-        assert isinstance(log.operation_time, datetime)
-        assert log.user.pk == self.no_rights_user.pk
-        assert isinstance(log.object_changes, dict)
+    def check_log(
+        self, log: AuditLog, operation_result: AuditLogOperationResult, user: User
+    ) -> None:
+        self.assertEqual(log.audit_object.object_id, self.cluster.pk)
+        self.assertEqual(log.audit_object.object_name, self.cluster.name)
+        self.assertEqual(log.audit_object.object_type, AuditObjectType.Cluster)
+        self.assertFalse(log.audit_object.is_deleted)
+        self.assertEqual(log.operation_name, "Cluster config log updated")
```

Comment:
I believe it should be "Cluster configuration updated" according to requirements

Reply:
I've noticed that some of records have "configuration" and some "config log"
Maybe there's a difference between create/restore config idk

Reply:
fixed